lib/honeybadger/plug.ex

```diff
@@ -7,6 +7,22 @@ defmodule Honeybadger.Plug do
       use Plug.ErrorHandler
       require Honeybadger
 
+      def call(conn, opts) do
+        try do
+          before_response = :erlang.monotonic_time
+
+          conn = super(conn, opts)
+
+          after_response = :erlang.monotonic_time
```

Comment:
I just remembered that monotonic_time is only supported by Erlang/OTP version 18. The Elixir community is pretty good about using the latest versions and Elixir 1.2 and all future versions have a hard dependency on OTP 18 anyway.